packages/cli/test/models/Dependency.test.js

```diff
@@ -29,7 +29,7 @@ contract('Dependency', function([_, from]) {
         it('throws error', function() {
           assertErrorMessage(
             () => Dependency.fromNameWithVersion('bildts-kcom'),
-            /Could not find a zos.json file/,
+            /Could not find a project.json file/,
```

Comment:
Should we change this error message to `an .openzeppelin/project.json file`?

Reply:
No opinion on that at all.